### PR TITLE
Moving linter and validations to gitlab

### PR DESCRIPTION
**Pull request by YuvHayun**
## Related Issues
fixes: [CIAC-5502](https://jira-hq.paloaltonetworks.local/browse/CIAC-5502)

## Description
Moving linter and validations to gitlab.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: .gitlab/ci/on-push.yml
Comment (JudahSchwartz) [inline .gitlab/ci/on-push.yml:76]: update this comment

File: .gitlab/ci/on-push.yml
Comment (JudahSchwartz) [inline .gitlab/ci/on-push.yml:70]: runs in circle for the on-push ^ correct this comment above

File: .gitlab/helper_functions.sh
Comment (JudahSchwartz) [inline .gitlab/helper_functions.sh:65]: why with quotes?

File: .gitlab/ci/on-push.yml
Comment (JudahSchwartz) [inline .gitlab/ci/on-push.yml:81]: what is this change?

File: .gitlab/ci/on-push.yml
Comment (JudahSchwartz) [inline .gitlab/ci/on-push.yml:73]: double check this

File: .gitlab/helper_functions.sh
Comment (YuvHayun) [reply .gitlab/helper_functions.sh:65]: @esharf added this, it's because of a failure we got "ambiguous redirect"

File: Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py
Comment (JudahSchwartz) [inline Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py:25]: delete before merge

File: .gitlab/ci/global.yml
Comment (YuvHayun) [reply .gitlab/ci/global.yml:258]: Thinking about this, we have to split the cases anyway..
here we're running lint on change since the last upload commit, in a pr we wish to run only on the changed files related to the current pr.

File: .gitlab/ci/on-push.yml
Comment (YuvHayun) [reply .gitlab/ci/on-push.yml:73]: entering in cases of on-push or contribution.

File: .gitlab/ci/on-push.yml
Comment (YuvHayun) [reply .gitlab/ci/on-push.yml:81]: entering in cases of on-push or contribution.

**Review by JudahSchwartz — APPROVED** [review]
please wait on other's approval, Eli and Dina

**Pull request merged**
